dotnet/src/Extensions/Experimental/Orchestration.Flow/Experimental.Orchestration.Flow.csproj

```diff
@@ -1,17 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <!-- THIS PROPERTY GROUP MUST COME FIRST -->
-    <AssemblyName>Microsoft.SemanticKernel.Planning.FlowPlanner</AssemblyName>
-    <RootNamespace>Microsoft.SemanticKernel.Planning.Flow</RootNamespace>
+    <AssemblyName>Microsoft.SemanticKernel.Experimental.OrchestrationOrchestrator</AssemblyName>
```

Comment:
Do we need the `Experimental` in the project name and namespace taking into account that it's the Experimental folder?

Reply:
Just a first proposal for now. Would love to get together with the crew to discuss options.

Reply:
My thinking was to help enable when it shows up in nuget it is very clear it is an extension|experimental. Someday we could have a meta package that would include all experimental extensions (Microsoft.SemanticKernel.Experimental) or you could pick individual pieces like *.Experimental.Orchestration.Flow (or similarly, all experimental orchestration libraries would be in *.Experimental.Orchestration).